Source/gui/SNTAboutWindowController.h

```diff
@@ -14,11 +14,5 @@
 
 #import <Cocoa/Cocoa.h>
 
-@interface SNTAboutWindowController : NSWindowController
-
-@property IBOutlet NSTextField *aboutTextField;
-@property IBOutlet NSButton *moreInfoButton;
-
-- (IBAction)openMoreInfoURL:(id)sender;
-
+@interface SNTAboutWindowController : NSWindowController <NSWindowDelegate>
```

Comment:
Has this file been formatted? I thought protocol subscriptions didn't have a space? `NSWindowController<NSWindowDelegate>`

Reply:
It didn't have a space and the linter complained, fix.sh put the space in.